src/core/Colors/Colors.baseStyles.tsx

```diff
@@ -17,20 +17,39 @@ export const baseStyles = withSuomifiTheme(
       flex-direction: column;
       justify-content: flex-end;
       float: left;
-      width: 192px;
-      height: 180px;
+      width: 208px;
+      height: 196px;
       max-width: 100%;
-      padding: ${theme.spacing.s};
       color: ${readableColor(color)};
-      background-color: ${color};
       border-bottom: 4px solid ${color};
       cursor: pointer;
       z-index: 2;
+      position: relative;
+      margin: 0;
 
       &:hover {
         border-bottom-color: ${readableColor(color)};
       }
 
+      figcaption {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-end;
+        position: absolute;
+        left: 8px;
+        bottom: 8px;
```

Comment:
Do we have design tokens that would suit this usecase?

Reply:
You mean the `left` and `bottom` values?

Like this?
```scss
left: ${theme.spacing.s};
bottom: ${theme.spacing.s};
```

Reply:
Yes, not sure if it's relevant here, but I like this better since those are defined.

Reply:
This change was made and committed.